lib/types/types.go

```diff
@@ -81,15 +83,46 @@ func NullDecoder(f reflect.Type, t reflect.Type, data interface{}) (interface{},
 	return data, nil
 }
 
+//TODO: something better that won't reuire so much boilerplate and casts for NullDuration values...
+
 // Duration is an alias for time.Duration that de/serialises to JSON as human-readable strings.
 type Duration time.Duration
 
 func (d Duration) String() string {
 	return time.Duration(d).String()
 }
 
+// ParseExtendedDuration is a helper function that allows for string duration
+// values containing days.
+func ParseExtendedDuration(data string) (result time.Duration, err error) {
```

Comment:
Why ?

Reply:
:roll_eyes: 
...
To be honest, because I needed it for `TestGetFixedLengthDuration()` :blush: :sweat_smile: And after I basically implemented it there, I decided that it might as well be added here, since we have our own `Duration` type and all... And I've personally been frustrated that I couldn't specify durations in days before in k6 and in Go, so why not

Reply:
It is way too much code that adds functionality which ... I doubt even you needed ;), so I would argue that you should scrap it ... 
If it was a couple of lines ... maybe, although I would argue that you should add weeks and months and years if you are going to add days ...

Reply:
> It is way too much code 

20 lines of code is way too much? :smile: and this is probably one of the more tested parts of k6 currently :smile: if I remove it, the test coverage will drop further :stuck_out_tongue_closed_eyes: 

> I would argue that you should add weeks and months and years if you are going to add days ...

maybe weeks, though that's rarely used for durations, but different months and different years don't have a constant number of days in them, so they're unsuitable

Reply:
:thinking: